src/timedate/timedated.c

```diff
@@ -502,8 +489,8 @@ static int unit_start_or_stop(UnitStatusInfo *u, sd_bus *bus, sd_bus_error *erro
                 "ss",
                 u->name,
                 "replace");
-        log_unit_full_errno_zerook(u, r < 0 ? LOG_WARNING : LOG_DEBUG, r,
-                                   "%s unit: %m", start ? "Starting" : "Stopping");
+        log_full_errno_zerook(r < 0 ? LOG_WARNING : LOG_DEBUG, r,
+                              "%s unit: %m", start ? "Starting" : "Stopping");
```

Comment:
I have my doubts if this is a good change. LOG_CONTEXT_PUSH sets the prefix, but that prefix and other metadata will be attached to any log line. So let's say `bus_call_method` does some general logging about the connection. It'd be quite confusing to have the message about `ntpd.service` attached there.